starter_code/src/movies.js

```diff
@@ -1,13 +1,40 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
-
+console.log(movies[0])
+
+    function stringToMinutes(string) {
```

Comment:
plase copy&paste my corrected (and shorter) version of this function from slack :)